src/Bindings/ManualBindings.cpp

```diff
@@ -1493,6 +1482,18 @@ static int tolua_cPluginManager_GetCurrentPlugin(lua_State * S)
 
 
 
+static int tolua_cPluginManager_GetPlugin(lua_State * tolua_S)
```

Comment:
I suppose we need different levels of "exists" - "exists and is loaded", "exists but don't care about status"